code/game/machinery/suit_storage_unit.dm

```diff
@@ -207,7 +216,6 @@
 	name = "radiation suit storage unit"
 	suit_type = /obj/item/clothing/suit/radiation
 	helmet_type = /obj/item/clothing/head/radiation
-	storage_type = /obj/item/geiger_counter
 
```

Comment:
Something seems to have gone a bit odd here in regards to the radiation refactor. This, and all the related radiation stuff needs to be set back to master.